### DISCUSSION TITLE
PullRequest Assignee

### DISCUSSION THREAD

**Discussion by BermudaLamb** [octokit/octokit.net#2675] [Q&A]
I see that there is a property for getting the Assignee of a PullRequest. However, I don't see any way to set the Assignee when creating a PullRequest. If there is can you please share how that is done?

**Comment by kfcampbell** [comment]
Hi! Does using the [AddAssignees](https://github.com/octokit/octokit.net/blob/main/Octokit/Clients/AssigneesClient.cs#L112) method present work?

**Comment by BermudaLamb** [comment]
No, I'm looking for a method signature of 
```c#
AddAssignees(long repositoryId, int number, PullRequestReviewRequest users)
```
and I don't see one in the Octokit.net nuget package.
